code/number-theory/Sieve.h

```diff
@@ -0,0 +1,23 @@
+const int N = 1e6;
```

Comment:
don't put in extra constants. They take too much space on paper. You can put them as inputs in each function for example.

Reply:
okay , i will do that. it is even better yes.